dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -106,6 +106,38 @@ abstract class DLCWallet extends Wallet with AnyDLCHDWalletApi {
     } yield updated
   }
 
+  private def updateFundingOutPoint(
+      contractId: ByteVector,
+      outPoint: TransactionOutPoint): Future[DLCDb] = {
```

Comment:
nit: Hmm... I guess I don't have a strong preference here, but do you think it's better to throw an exception rather than just returning `Future[Option[DLCDb]]`? Same applies with the `updateClosingTxId` method.

Reply:
I think this is better, we are only going to be calling this function inside of other functions that should have already verified that the `contractId` exists. Making this an option will just make things unnecessarily complicated

Reply:
Ok fair enough, it sees we just return a `T` everywhere else rather than an `Option[T]` when we use the term `update`